README.md

```diff
@@ -10,47 +10,47 @@ Wechaty is a Bot Framework for Wechat **Personal** Account which can help you cr
 [![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-blue.svg)](https://github.com/ellerbrock/typescript-badges/)
 [![Repo Size](https://reposs.herokuapp.com/?path=Chatie/wechaty)](https://github.com/chatie/wechaty)
 
-:octocat: <https://github.com/chatie/wechaty>  
-:beetle: <https://github.com/chatie/wechaty/issues>  
-:book: <https://github.com/chatie/wechaty/wiki>  
-:whale: <https://hub.docker.com/r/zixia/wechaty>  
+:octocat: <https://github.com/chatie/wechaty>
+:beetle: <https://github.com/chatie/wechaty/issues>
+:book: <https://github.com/chatie/wechaty/wiki>
+:whale: <https://hub.docker.com/r/zixia/wechaty>
 
 ## Voice of the Developer
 
-> I ... fall in love with it. It’s really easy to get started to make your own wechatbot. [link](http://blog.chatie.io/2017/05/25/use-interval-in-wechaty.html)  
+> I ... fall in love with it. It’s really easy to get started to make your own wechatbot. [link](http://blog.chatie.io/2017/05/25/use-interval-in-wechaty.html)
 > -- @kungfu-software, Founder of Kungfu Software
 
-> "Wechaty is a great solution, I believe there would be much more users recognize it." [link](https://github.com/chatie/wechaty/pull/310#issuecomment-285574472)  
-> -- @Gcaufy, Tencent 
+> "Wechaty is a great solution, I believe there would be much more users recognize it." [link](https://github.com/chatie/wechaty/pull/310#issuecomment-285574472)
+> -- @Gcaufy, Tencent
 
-> "The best wechat SDK I have seen in Github!" [link](https://github.com/chatie/wechaty/issues/8#issuecomment-228971491)  
+> "The best wechat SDK I have seen in Github!" [link](https://github.com/chatie/wechaty/issues/8#issuecomment-228971491)
 > -- @JasLin, BotWave CTO
 
-> "Wechaty简单的接口...和Docker化的封装...绝对是一个不错的选择" [link](http://mp.weixin.qq.com/s/o-4VMcAMz0K8yJVdNaUXow)  
+> "Wechaty简单的接口...和Docker化的封装...绝对是一个不错的选择" [link](http://mp.weixin.qq.com/s/o-4VMcAMz0K8yJVdNaUXow)
 > -- @shevyan, Ghost Cloud CEO
 
-> "Wechaty is great." [link](https://github.com/chatie/wechaty/issues/36#issuecomment-251708382)  
+> "Wechaty is great." [link](https://github.com/chatie/wechaty/issues/36#issuecomment-251708382)
 > -- @Samurais, SnapLingo Director of Engineering
 
-> "最好的微信开发库" [link](http://weibo.com/3296245513/Ec4iNp9Ld?type=comment)  
```

Comment:
Do NOT strip the spaces at the end of the line in Markdown file, it's meaningful.